docs/tracks.md

```diff
@@ -21,16 +21,16 @@ Simple enough, right?
 
 ## Social Verification
 
-- User signs nonce (message) using wallet
-- Tweet out the signature tagging [`@authdeck`](https://twitter.com/authdeck)
+- User signs nonce (message) using their wallet
+- Tweets out the signature tagging [`@authdeck`](https://twitter.com/authdeck)
```

Comment:
Can you change this to "User tweets ..." ? @KortaPo

Reply:
Okay I will, sorry for the late reply.